azure_sdk/lib/Profile_Azure_Stack/module_definition.rb

```diff
@@ -0,0 +1,8 @@
+# encoding: utf-8
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+
+module Azure end
+module Azure::Profiles end
+module Azure::Profiles::Management end
+module Azure::Profiles::Management::Profile_Azure_Stack end
```

Comment:
I do not think this will work. There are profiles with name Profile_2010_10_01. Now, if we start removing the '_',
then it must be references as Profile20101001. We would want to keep things consistent. So, retaining '_' here.